modules/dnn/src/layers/eltwise_layer.cpp

```diff
@@ -466,6 +473,13 @@ class EltwiseLayerImpl CV_FINAL : public EltwiseLayer
     {
         auto context = reinterpret_cast<csl::CSLContext*>(context_);
 
+        if(variableChannels)
```

Comment:
`variableChannels` has been removed by #16088

Perhaps check here should looks like
```
channelsMode == ELTWISE_CHANNNELS_INPUT_0
    || channelsMode == ELTWISE_CHANNNELS_INPUT_0_TRUNCATE
```

and else branch should ensure that `channelsMode == ELTWISE_CHANNNELS_SAME`.